tests/test_introduction_to_classes.py

```diff
@@ -58,9 +59,18 @@ def test_Fruit_bite_it_negative(input_neg):
 ])
 def test_is_edible(input_val, expected):
 
-    orange = Fruit(test_name, test_color, test_flavor, input_val, test_weight)
-    assert orange.is_edible(input_val) == expected
+    orange = Fruit(test_name, test_color, test_flavor, poisonous=input_val, weight=test_weight)
+    assert orange.is_edible() == expected
 
+@pytest.mark.parametrize("color_val,expected", [
+    ("yellow", "yellow"),
+    ("green", "green"),
+    ("red", "red")
```

Comment:
there is no need to have expected, because `expected` is just a copy of `color_val`. so get rid of expected. just use. color_val to validate.

Reply:
I do not understand how to do it right. Remove the expected values ​​themselves or ?? Or instead of the values ​​expected to substitute color_val?

Reply:
OK
here is what i wanted from you:
```python
@pytest.mark.parametrize("color_val", ["yellow", "green", "red"])
def test_get_color(color_val, expected):
    Lemon = Fruit(test_name, color_val, test_flavor, test_poisonous, test_weight)
    assert Lemon.get_color() == color_val
```